src/screens/Profiles/components/ImageCard/component.js

```diff
@@ -1,63 +1,123 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { View } from 'react-native';
+import { View, TouchableOpacity, ViewPropTypes, Text } from 'react-native';
 import { StyledText } from 'kitsu/components/StyledText';
-import { MaskedImage } from 'kitsu/screens/Profiles/components/MaskedImage';
-import { MediaCard } from 'kitsu/screens/Profiles/components/MediaCard';
-import { scenePadding } from 'kitsu/screens/Profiles/constants';
+import { ProgressiveImage } from 'kitsu/components/ProgressiveImage';
+import { cardSize } from 'kitsu/screens/Profiles/constants';
+import LinearGradient from 'react-native-linear-gradient';
 import { styles } from './styles';
 
 const paddingOptions = {
-  landscapeLarge: scenePadding,
-  filled: scenePadding,
-  landscape: scenePadding / 2,
-  portrait: scenePadding / 2,
-  portraitLarge: scenePadding,
-  square: scenePadding / 2,
-  thumbnail: scenePadding / 2,
+  landscapeLarge: 14,
+  filled: 14,
+  landscape: 10,
+  portrait: 8,
+  portraitLarge: 10,
+  square: 10,
+  thumbnail: 10,
+};
+
+const TextView = ({ variant, title, subtitle, boldTitle, centerTitle, noMask }) => {
+  const titleSize = variant === 'landscapeLarge' ? 'default' : 'xsmall';
+  const subtitleSize = variant === 'landscapeLarge' ? 'small' : 'xxsmall';
+
+  return (title || subtitle) && (
+    <View style={[styles.anchorBottom, { height: '60%' }]}>
+      { !noMask && <LinearGradient colors={['transparent', 'rgba(0,0,0,0.9)']} style={styles.linearGradient} />}
+      <View style={[styles.anchorBottom, { padding: paddingOptions[variant] }]}>
+        {subtitle && <StyledText color="lightGrey" size={subtitleSize} numberOfLines={1}>{subtitle}</StyledText>}
```

Comment:
try passing an empty string to the subtitle, it should crash the app. exactly the same reason with the above